std_msgs/String bin
---
string[] bin_contents
